LEMS/LEMS_GravCalcs.py

```diff
@@ -353,7 +375,7 @@ def LEMS_GravCalcs(gravinputpath,aveinputpath,timespath,energypath,gravoutputpat
             endtime=gravval[endname]         #variable value (string) of the phase end time from the phase times input file
             duration=timeperiod(starttime,endtime)  #phase length in minutes
         
-        for train in ['A','B', 'SB3002', 'SB3001', 'Possum2']: #for each grav flow train
+        for train in ['A','B', 'SB3002', 'SB3001', '3009', 'Possum2']: #for each grav flow train

```

Comment:
the 3009 looks like it should be SB3009 so I updated that in the SB3015/16 commit.